Kickstarter-iOS/Views/Controllers/RewardsCollectionViewController.swift

```diff
@@ -3,12 +3,6 @@ import KsApi
 import Library
 import Prelude
 
-private enum Layout {
-  enum Card {
-    static let width: CGFloat = 249
```

Comment:
Moved to a static var so that it's accessible elsewhere. Could be moved to a `CheckoutConstants` `enum`?